frontend/libs/studio-components/src/components/StudioAvatar/StudioAvatar.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { type ReactElement } from 'react';
+import classes from './StudioAvatar.module.css';
+import { PersonCircleIcon } from '@studio/icons';
+
+type ImageDetails = {
+  src: string;
+  alt: string;
+  title: string;
+};
+
+export type StudioAvatarProps = {
+  imageDetails?: ImageDetails;
+};
+
+export const StudioAvatar = ({ imageDetails }: StudioAvatarProps): ReactElement => {
+  if (!imageDetails) {
+    return <PersonCircleIcon className={classes.avatar} />;
+  }
+  return (
+    <img
+      alt={imageDetails.alt}
+      title={imageDetails.title}
+      className={classes.avatar}
+      src={imageDetails.src}
+    />
+  );
+};
```

Comment:
Ref. the previous comment, we can "spread" the `avatarElement`, since it's using the native element instead. What do you think? 😊 

```suggestion
export const StudioAvatar = ({ avatarElement }: StudioAvatarProps): ReactElement => {
  if (!avatarElement ) {
    return <PersonCircleIcon className={classes.avatar} />;
  }
  
  return (
    <img
       {...avatarElement}
      className={cn(classes.avatar, avatarElement.className)}
    />
  );
};
```